### ISSUE TITLE
Sticky Note does not go in Library

### ISSUE THREAD

**Issue by mtg1986** [IBVSoftware/CollaBoard-Feedback#302]
1.Move a sticky not to the library
2.Instead of having the sticky note moving into the library nothing happens. We left the project and reloaded the project. After that we could move it into the library. 
I know it is a very large project and we uploaded a lot of stuff. We also got an "Out of Ink ressource Error".

-->Will be difficult to reproduce, but we could do it in the same project. Ask Tamara she can reproduce it if needed. 

Tested on 
HP Spectre
Version 1.3.106
Project: 8932, grro2

![stickydoesnotgoin](https://user-images.githubusercontent.com/19570958/30030297-f3880658-918c-11e7-9eea-ae7c99928d21.JPG)

**Comment by mtg1986** [comment]
![outofink](https://user-images.githubusercontent.com/19570958/30030323-07b5cd90-918d-11e7-8609-f624cbb7f3e6.JPG)

**Comment by corradocavalli** [comment]
I might be wrong but tiles that are in edit mode cannot be moved into libraries by default) that's why you can't initially drag them into.
The message you see is because we have a limited number of tiles that ca be simultaneously be in edit mode.


On 4 Sep 2017, at 10:21, Michael Görög <notifications@github.com<mailto:notifications@github.com>> wrote:


[outofink]<https://user-images.githubusercontent.com/19570958/30030323-07b5cd90-918d-11e7-8609-f624cbb7f3e6.JPG>

**Comment by mtg1986** [comment]
Hi Corrado,
please have a look at the picture. The Sticky Note Tamara dropped was not in edit mode anymore. As it was working after reloading the project I think this is not a high priority issue. 

Do you know the exact number of tiles that can be simultaniously in edit mode?

**Comment by gsantopaolo** [comment]
Hi guys, this seems another bug that we have to get ride of during testing week.
Please do not close this until we can reproduce, fix and say that is then working

thanks

**Comment by corradocavalli** [comment]
If was not in edit mode it might be a bug, please try to provide a reproducible scenario.
Technically is 127, but value changes depending on resources, it would say.more or less around 100.

On 4 Sep 2017, at 13:21, Gian Paolo Santopaolo <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi guys, this seems another bug that we have to get ride of during testing week.
Please do not close this until we can reproduce, fix and say that is then working

thanks

**Comment by corradocavalli** [comment]
Tamara helped me understanding what is the wrong behavior reported.

In this particular project when you "throw" a tile over a library it doesn't get 'eaten up' by the library but instead it just stop above.
The expected use of drag dropping tile into libraries is indeed **not to throw them** but **manually drag them over and release them when frame becomes azure**, the fact that it goes into even when thrown is an inherited behavior due to internal handling of tile inertia but it is not reliable and in fact it depends on how much processing is involved during drag-drop operation.
The 'official' manual drag-drop always work, tested by both me and Tamara.
@gsantopaolo How should we proceed? My opinion is to avoid the tile to go automatically into libraries since, as we saw many times, most often lib tile catch up tile that have been inadvertently been moved.
